src/health_manager.py

```diff
@@ -100,32 +101,45 @@ def start_monitor(self, run_thread: Thread):
             if not is_loading_black_roi:
                 # check health
                 health_percentage = self.get_health(img)
+                last_rejuv_drink = time.time() - self._last_rejuv
                 last_drink = time.time() - self._last_health
-                if health_percentage < self._config.char["take_health_potion"] and last_drink > 3.5:
-                    self._drink_poition(img, "health")
-                    self._last_health = time.time()
+
                 # give the chicken a 6 sec delay to give time for a healing pot and avoid endless loop of chicken
-                elif health_percentage < self._config.char["chicken"] and (time.time() - start) > 6:
+                if health_percentage < self._config.char["chicken"] and (time.time() - start) > 6:
```

Comment:
We have higher prio on chicken vs rejuv potion. Shouldnt it be the other way around. Bascially, try to take rejuv potion, only if you don't have anymore, do chicken?